src/App.js

```diff
@@ -27,6 +27,20 @@ function App() {
   const [uid, setUid] = useState(null);
   const [user, setUser] = useState(null);
   const [themeType, setThemeType] = useStorage("theme", "light");
+  const [customLightTheme, setCustomLightTheme] = useState(lightTheme);
+  const [customDarkTheme, setCustomDarkTheme] = useState(darkTheme);
+  const [customColors] = useStorage("customColors", null);
+
+  useEffect(() => {
+    if (customColors) {
+      if (customColors.light) {
+        setCustomLightTheme(createMuiTheme({...lightTheme, setCard: {...lightTheme.setCard, ...customColors.light}}));
```

Comment:
I believe the `createMuiTheme` call is redundant here. Doesn't this function just merge in missing properties in the theme?

Reply:
You're right, I had assumed createMuiTheme had some necessary side effects, but it seems not. Will simplify.